Assets/Mirror/Runtime/NetworkConnection.cs

```diff
@@ -259,20 +259,13 @@ public virtual bool TransportSend(int channelId, byte[] bytes, out byte error)
 
         internal void AddOwnedObject(NetworkIdentity obj)
         {
-            if (clientOwnedObjects == null)
-            {
-                clientOwnedObjects = new HashSet<uint>();
-            }
+            clientOwnedObjects = clientOwnedObjects ?? new HashSet<uint>();
```

Comment:
let me look into this. perhaps we can just create clientobjects once and use clear from that point on